packages/server/src/util/test-utils/roster.test-generator.ts

```diff
@@ -0,0 +1,39 @@
+import { CustomRosterColumn } from '../../api/roster/custom-roster-column.model';
+import { Unit } from '../../api/unit/unit.model';
+import { Roster } from '../../api/roster/roster.model';
+import { uniqueEdipi } from './unique';
+import { getCustomRosterColumnTestData } from './custom-roster-column.test-generator';
+import { getOrgTestData } from './org-test-generator';
+import { getAdminUserTestData } from './user.test-generator';
+import { getUnitsTestData } from './unit.test-generator';
+import { getReportSchemaTestData } from './report-schema.test-generator';
+
+export function getRosterTestData(numRosterEntries: number, customColumn: CustomRosterColumn, units: Unit[], numUnits: number) {
+  const rosterEntries: Roster[] = [];
+  for (let i = 0; i < numRosterEntries; i++) {
+    const customColumns: any = {};
+    customColumns[customColumn.name] = `custom column value`;
+    const edipi = uniqueEdipi();
```

Comment:
uniqueEdipi is thorn in our side (as you already have pointed out) and even more annoying that we need to parse it as integer at times like below. Let's talk with Roel to see if this method can be adjusted.

Reply:
I think it is fine as is,  we have a bigger fish to fry. Optionally, we can remove the parse Int call and append the edipi as is or generate better names. This is out of scope I think for now.